Update integration_test README (#3824)
flutter/plugins: packages/integration_test/README.md

# integration_test (moved)

## MOVED

> This package has been [moved to the Flutter

Reviewer: I'm not convinced this is clearer; "moved" isn't generally a meaningful term for clients of a package (since where the source lives is generally irrelevant). The version on pub, where users are likely to be reading this, is deprecated.

Author: The version on Pub is deprecated, but the package isn't. We don't want to give the impression that integration testing should be done some other way. @sfshaza2 do you have any input here?

Reviewer 2: I made some suggested changes to the text.

> This package has been [moved to the Flutter

Reviewer 2: ```suggestion
This package has [moved to the Flutter
```

> SDK](https://github.com/flutter/flutter/tree/master/packages/integration_test).

Reviewer 2: ```suggestion
SDK](https://github.com/flutter/flutter/tree/master/packages/integration_test),
and the pub.dev version is deprecated.
```

> Starting with Flutter 2.0, it should be included as:

Reviewer 2: ```suggestion
As of Flutter 2.0, include it in your pubspec's
dev dependencies section, as follows:
```
